programs/locked-voter/src/instructions/merge_partial_unstaking.rs

```diff
@@ -0,0 +1,67 @@
+use crate::*;
+
+/// Accounts for [voter::merge_partial_unstaking].
+#[derive(Accounts)]
+pub struct MergePartialUnstaking<'info> {
+    /// [Locker].
+    #[account(mut)]
+    pub locker: Box<Account<'info, Locker>>,
+
+    /// [Escrow].
+    #[account(mut, has_one = locker, has_one = owner)]
+    pub escrow: Box<Account<'info, Escrow>>,
+
+    /// The [Escrow] that is being closed.
```

Comment:
There are some other places with incorrect comment too. Could you update it?

Reply:
Fixed